packages/fiori/src/FlexibleColumnLayout.ts

```diff
@@ -0,0 +1,815 @@
+	*  - <code>startArrowLeftText</code>: the text that the first arrow (between the <code>begin</code> and <code>mid</code> columns) will have when pointing to the left
+	*  - <code>startArrowRightText</code>: the text that the first arrow (between the <code>begin</code> and <code>mid</code> columns) will have when pointing to the right
+	*  - <code>endArrowLeftText</code>: the text that the second arrow (between the <code>mid</code> and <code>end</code> columns) will have when pointing to the left
+	*  - <code>endArrowRightText</code>: the text that the second arrow (between the <code>mid</code> and <code>end</code> columns) will have when pointing to the right
+	*  - <code>startArrowContainerAccessibleName</code>: the text that the first arrow container (between the <code>begin</code> and <code>mid</code> columns) will have as <code>aria-label</code>
+	*  - <code>endArrowContainerAccessibleName</code>: the text that the second arrow container (between the <code>mid</code> and <code>end</code> columns) will have as <code>aria-label</code>
+	*
+	* @type {object}
+	* @name sap.ui.webc.fiori.FlexibleColumnLayout.prototype.accessibilityTexts
+	* @public
+	* @since 1.0.0-rc.11
+	*/
+	@property({ type: Object })
+	accessibilityTexts!: AccessiblilityTexts;
+
+	/**
+	* An object of strings that defines additional accessibility roles for further customization.
+	*
+	* It supports the following fields:
+	*  - <code>startColumnRole</code>: the accessibility role for the <code>startColumn</code>
+	*  - <code>startArrowContainerRole</code>: the accessibility role for the first arrow container (between the <code>begin</code> and <code>mid</code> columns)
+	*  - <code>midColumnRole</code>: the accessibility role for the <code>midColumn</code>
+	*  - <code>endArrowContainerRole</code>: the accessibility role for the second arrow container (between the <code>mid</code> and <code>end</code> columns)
+	*  - <code>endColumnRole</code>: the accessibility role for the <code>endColumn</code>
+	*
+	* @type {object}
+	* @public
+	* @name sap.ui.webc.fiori.FlexibleColumnLayout.prototype.accessibilityRoles
+	* @since 1.1.0
+	*/
+	@property({ type: Object })
+	accessibilityRoles!: AccessibilityRoles;
+
+	/**
+	* Defines the component width in px.
+	*
+	* @type {sap.ui.webc.base.types.Float}
+	* @defaultvalue 0
+	* @private
+	*/
+	@property({ validator: Float, defaultValue: 0 })
+	private _width!: number;
+
+	/**
+	* Defines the effective columns layout,
+	* based on both the <code>layout</code> property and the screen size.
+	* Example: [67%, 33%, 0], [25%, 50%, 25%], etc.
+	*
+	* @type {object}
+	* @defaultvalue undefined
+	* @private
+	*/
+	@property({ type: Object, defaultValue: undefined })
+	private _columnLayout?: ColumnLayout;
+
+	/**
+	* Defines the visible columns count - 1, 2 or 3.
+	*
+	* @type {sap.ui.webc.base.types.Integer}
+	* @defaultvalue 1
+	* @private
+	*/
+	@property({ validator: Integer, defaultValue: 1 })
```

Comment:
defaultValue = 0 because you are changing the current behaviour. You can change it to 1 if you've contacted the corresponding team to validate it.